### PR TITLE
Update DeviceRegistration.md

### PR DESCRIPTION
**Pull request by jaimeo**
My edits and a few new questions. It seems I don't have write access to your branch, so we'll have to PR from this fork/branch....

### CODE REVIEW

File: microsoft-365/managed-desktop/service-description/DeviceRegistration.md
Comment (jwmurraycmu) [inline microsoft-365/managed-desktop/service-description/DeviceRegistration.md:40]: Can we maybe put this note before the Hardware Hash? I'm worried folks scanning the list may miss this important piece.

File: microsoft-365/managed-desktop/service-description/DeviceRegistration.md
Comment (jwmurraycmu) [inline microsoft-365/managed-desktop/service-description/DeviceRegistration.md:50]: Oh good catch. We're adding support to the website that will just take this format as is. @Magana - you think that will land in the short term? If not, we should totally add a step to merge the files.

File: microsoft-365/managed-desktop/service-description/DeviceRegistration.md
Comment (jwmurraycmu) [inline microsoft-365/managed-desktop/service-description/DeviceRegistration.md:65]:
```suggestion
>[!TIP]
``` (missed the closing bracket)

File: microsoft-365/managed-desktop/service-description/DeviceRegistration.md
Comment (jwmurraycmu) [inline microsoft-365/managed-desktop/service-description/DeviceRegistration.md:68]: For the powershell blocks do we want to use "```"?

File: microsoft-365/managed-desktop/service-description/DeviceRegistration.md
Comment (jwmurraycmu) [inline microsoft-365/managed-desktop/service-description/DeviceRegistration.md]: Thanks for this. From now on I'll use the MD style tables - appreciate the translation this time.

File: microsoft-365/managed-desktop/service-description/DeviceRegistration.md
Comment (jaimeo) [reply microsoft-365/managed-desktop/service-description/DeviceRegistration.md:68]: Typically for single (inline) PS, we just use the single backtick. The ``` is for whole blocks or scripts.

File: microsoft-365/managed-desktop/service-description/DeviceRegistration.md
Comment (jwmurraycmu) [inline microsoft-365/managed-desktop/service-description/DeviceRegistration.md:167]: Nice catch - it is a bit odd, but accurate. The reason is that for customers we use the Hardware Hash to look up the device, so the other values aren't really needed. For PArtners we have no hash, and so we use the other three values instead. Since we're using them to do a look-up, they need to be **perfect**

File: microsoft-365/managed-desktop/service-description/DeviceRegistration.md
Comment (jaimeo) [reply microsoft-365/managed-desktop/service-description/DeviceRegistration.md]: No worries! HTML can hork things up, so we don't use it unless there is just no other option (and there are cases where that is true).

File: microsoft-365/managed-desktop/service-description/DeviceRegistration.md
Comment (jaimeo) [reply microsoft-365/managed-desktop/service-description/DeviceRegistration.md:167]: Ah so--OK, that'll be something I'll really emphasize (in both places). That way in case someone reads both they won't get confused or trip over the inconsistency (HOPEFULLY).

File: microsoft-365/managed-desktop/service-description/DeviceRegistration.md
Comment (jaimeo) [reply microsoft-365/managed-desktop/service-description/DeviceRegistration.md:40]: Sorry, which Note exactly? I mean probably yes, but I want to be sure I'm looking at the right one.

File: microsoft-365/managed-desktop/service-description/DeviceRegistration.md
Comment (jwmurraycmu) [inline microsoft-365/managed-desktop/service-description/DeviceRegistration.md:150]: RE: HTML - rgr. I'm good with MD format from now on.

RE: not saying please, I'm good with that.

RE: Hardware Hash invalid - good with your edit

RE: model/man - that error message should only come up for Partners (see below). If we have a separate topic, maybe we remove it from this table since users should never see it.

RE: yeah, there's a de-registration action. However, we recently decided not to allow customers to do that on their own in v1. We can remove those rows from the table for now (and chat more about deregistration offline).

RE: explaining export - we should add that! Totally a miss.

**Review by jwmurraycmu — CHANGES_REQUESTED** [review]
Added a bunch of comments. Let's finish this round as a PR and then I'll just get you guys write access directly (or we can figure out a "proper" editing branch solution)

File: microsoft-365/managed-desktop/service-description/DeviceRegistration.md
Comment (jaimeo) [reply microsoft-365/managed-desktop/service-description/DeviceRegistration.md:150]: "Invalid": I meant in the error itself, which of course would be code change. The doc of course will quote whatever the UI or error string says. The "please" thing is actually MS policy--it's a globalization/cultural thing.

File: microsoft-365/managed-desktop/service-description/DeviceRegistration.md
Comment (jwmurraycmu) [reply microsoft-365/managed-desktop/service-description/DeviceRegistration.md:68]: rgr that

File: microsoft-365/managed-desktop/service-description/DeviceRegistration.md
Comment (jwmurraycmu) [reply microsoft-365/managed-desktop/service-description/DeviceRegistration.md:40]: The one that reads "Unlike the other values, the hardware hash..."

File: microsoft-365/managed-desktop/service-description/DeviceRegistration.md
Comment (jwmurraycmu) [reply microsoft-365/managed-desktop/service-description/DeviceRegistration.md:150]: hey @Magana, can you integrate the feedback on the error text for hardware hash?

File: microsoft-365/managed-desktop/service-description/DeviceRegistration.md
Comment (jwmurraycmu) [reply microsoft-365/managed-desktop/service-description/DeviceRegistration.md:167]: Rgr, good plan. thanks man!

File: microsoft-365/managed-desktop/service-description/DeviceRegistration.md
Comment (Magana) [reply microsoft-365/managed-desktop/service-description/DeviceRegistration.md:50]: For now, let's add the step to merge the files, and we can remove the instructions once we're comfortable with other formats

File: microsoft-365/managed-desktop/service-description/DeviceRegistration.md
Comment (Magana) [reply microsoft-365/managed-desktop/service-description/DeviceRegistration.md:150]: Sure thing!

File: microsoft-365/managed-desktop/service-description/DeviceRegistration.md
Comment (jwmurraycmu) [reply microsoft-365/managed-desktop/service-description/DeviceRegistration.md:50]: roger that. @jaimeo - once we merge I'll add some draft.

File: microsoft-365/managed-desktop/service-description/DeviceRegistration.md
Comment (jwmurraycmu) [reply microsoft-365/managed-desktop/service-description/DeviceRegistration.md:150]: Sweet, @jaimeo - let me know what you think about having a separate table for PArtners. Also, let's remove the de-registration rows completely for now.

Comment (jaimeo) [reply microsoft-365/managed-desktop/service-description/DeviceRegistration.md:150]: Sorry, thought I mentioned that: yes of course since we'll have a completely separate Partners topic, that topic will have its own version of the table. I think even if there is some duplication of the errors in the table(s), it's worth doing b/c once we vector a reader down a particular track, it's best to keep them on that track w/o bouncing over to topics on the other track, if you see what I mean.

**Pull request closed without merging**
